src/index.ts

```diff
@@ -73,39 +74,22 @@ async function assetToAccount(
   );
 }
 
-async function createCaip10Link(
-  account: AccountID, 
-  timestamp: number, 
-  ceramic: CeramicApi
-): Promise<string | null> {
-  const doc = await ceramic.createDocument('caip10-link', {
-    metadata: {
-      family: 'caip10-link',
-      controllers: [AccountID.format(account)]
-    }
-  });
-  if (timestamp) {
-    const docAtTime = await ceramic.loadDocument(doc.id, { atTime: timestamp });
-    return docAtTime?.content;
-  }
-  return doc?.content;
-}
-
 /**
  * Creates CAIP-10 links for each account to be used as controllers. 
  * Since there may be many owners for a given NFT (only ERC1155 for now), there can be many
  * controllers of that DID document.
  */
 async function accountsToDids(
   accounts: AccountID[], 
-  timestamp: number, 
+  // NOTE: Opts: { atTime: timestamp } no longer applicable for create?
```

Comment:
Can you elaborate on the issue you saw trying to use atTime with Caip10Link?  What code did you try and what error did you see?

Reply:
> Can you elaborate on the issue you saw trying to use atTime with Caip10Link? What code did you try and what error did you see?

It may just be a type error but it appears as though the `CreateOps` type extends `BasicLoadOps` instead of `LoadOps` which does not contain the `atTime` property.

```
interface BasicLoadOpts {
    sync?: SyncOptions;
    syncTimeoutSeconds?: number;
}
export interface LoadOpts extends BasicLoadOpts {
    atTime?: number;
}
export interface UpdateOpts {
    anchor?: boolean;
    publish?: boolean;
}
export interface CreateOpts extends UpdateOpts, BasicLoadOpts {
}
```

Reply:
Ah yes, sorry about that, that seems like an oversight. Fix is here: https://github.com/ceramicnetwork/js-ceramic/pull/1495

Reply:
Oh nice, no worries. Thanks for jumping on it!